packages/interface-compliance-tests/src/mocks/connection.ts

```diff
@@ -76,24 +74,22 @@ class MockConnection implements Connection {
       throw new Error('protocols must have a length')
     }
 
-    if (this.status !== STATUS.OPEN) {
```

Comment:
Why did we remove the string constant? I see this is re-used in multiple places

Reply:
For consistency with types used elsewhere, e.g.

```ts
export type Direction = 'inbound' | 'outbound'
```

```ts
export type PeerIdType = 'RSA' | 'Ed25519' | 'secp256k1'
```

```ts
export type PendingDialStatus = 'queued' | 'active' | 'error' | 'success'
```

etc

Reply:
If we want to reduce duplication these should probably be enums rather than strings, but that can be done in a follow-up PR.